snapshot/policy/retention_policy.go

```diff
@@ -147,7 +150,17 @@ func (r *RetentionPolicy) getRetentionReasons(i int, s *snapshot.Manifest, cutof
 		{cutoff.hourly, s.StartTime.Format("2006-01-02 15"), "hourly", r.KeepHourly},
 	}
 
+	if r.KeepMinDays != nil {
+		if !s.StartTime.Before(cutoff.minDays) && *r.KeepMinDays > *newOptionalInt(0) {
+			keepReasons = []string{"Inside min retention days"}
+		}
+	}
+
 	for _, c := range cases {
+		if r.KeepMinDays != nil && *r.KeepMinDays > 0 {
```

Comment:
you wouldn't need this if you simply remove entry from `cases` and handle it separately.